pkg/ssh/ssh.go

```diff
@@ -182,8 +182,18 @@ func RunSSHUpgradeAvalanchego(host *models.Host, network models.Network, avalanc
 	if err != nil {
 		return err
 	}
-
-	if err := docker.ComposeSSHSetupNode(host, network, avalancheGoVersion, withMonitoring, publicAccessToHTTPPort); err != nil {
+	if err := docker.ComposeOverSSH("Compose Node",
```

Comment:
Looks like we can rid of the `network` parameter here now if it's no longer used?